### ISSUE TITLE
Unable to use VisualVM

### ISSUE THREAD

**Issue by joseairosa** [trinidad/trinidad#68]
Hello,

I've been trying to set up VisualVM with my jruby app, however I ran into the following issue.

If I try to run `sudo jmap -histo:live 9032`, 9032 being my app pid, I get the following error:

"9032: Unable to open socket file: target process not responding or HotSpot VM not loaded
The -F option can be used when the target process is not responding"

I'm getting these errors all over VisualVM as well.

After some careful search and read I've found this article: http://www.permeance.com.au/web/terry.mueller/home/-/blogs/365867;jsessionid=78761E367DAA498849D5694EC586AC4F

Basically it says that the error might be related with the explicit declaration of tmpdir. 

Is there anyway we can access these options via trinidad? And do you have any other suggestions to solve this issue?

Thank you!
